kong.conf.default

```diff
@@ -178,6 +178,10 @@
 # See http://nginx.org/en/docs/http/ngx_http_realip_module.html#real_ip_recursive
 # for a description of this directive.
 
+#error_default_type = text/plain  # Default response format to use when the
+                                  # 'Accept' header is missing and Nginx
+                                  # is returning an error for the request.
```

Comment:
This should also document which values are accepted if they are validated via an `enum` in the config loader.